content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -249,18 +297,18 @@ Content-Type: application/json
 }
 ```
 
-Table responses are available beginning in version 1.10 of the kube-apiserver. As such, not all API resource types will support a Table response, specifically when using a client against older clusters. Clients that must work against all resource types, or can potentially deal with older clusters, should specify multiple content types in their `Accept` header to support fallback to non-Tabular JSON:
+Not all API resource types will support a Table response; for example, with a CustomResourceDefinition or an APIService that extends the core Kubernetes API. Clients that must work against all resource types, should specify multiple content types in their `Accept` header to support fallback to non-Tabular JSON:
```

Comment:
nit, possible edit:
Not all API resource types support a Table response such as the CustomResourceDefinition or APIService resources. Clients that must work against all resource types should specify multiple content types in their `Accept` header to support fallback to non-Tabular JSON:

Reply:
Could you repeat this sentence (from below):
Not all API resource types support [Protobuf], specifically those defined via CustomResourceDefinitions or those that are API extensions.

Reply:
Addressed, I think.